fga/cache/cachedFgaTenantStore.go

```diff
@@ -0,0 +1,88 @@
+package cache
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"slices"
+	"time"
+
+	"github.com/hashicorp/golang-lru/v2/expirable"
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"google.golang.org/grpc/status"
+)
+
+type CachedFgaTenantStore struct {
```

Comment:
[Proposal]
We agreed on this name, but now I see better option:
lets call package `store` and this struct Service, so when imported it will be like this:
```
import "github.com/openmfp/golang-commons/fga/strore"

s := store.Service{}
s.GetStoreIDForTenant(....)
```

Reply:
done